app/assets/images/payment_icons/ask.svg

```diff
@@ -0,0 +1,27 @@
+<svg viewBox="0 0 38 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" role="img" width="38" height="24" aria-labelledby="pi-ask">
+    <title id="pi-ask">ASK</title>
+    <defs>
+        <path d="M3 0C1.3 0 0 1.3 0 3C0 4.8 0 19.2 0 21C0 22.7 1.4 24 3 24C6.2 24 31.8 24 35 24C36.7 24 38 22.7 38 21C38 19.2 38 4.8 38 3C38 1.3 36.6 0 35 0C28.6 0 6.2 0 3 0Z" id="a3iu3fiAv6"></path>
+        <path d="M35 1C36.1 1 37 1.9 37 3C37 4.8 37 19.2 37 21C37 22.1 36.1 23 35 23C31.8 23 6.2 23 3 23C1.9 23 1 22.1 1 21C1 19.2 1 4.8 1 3C1 1.9 1.9 1 3 1C5.13 1 15.8 1 35 1" id="b2E3Ggvq3e"></path>
+        <path
+            d="M6.37 15.74L5.47 18.55L3.05 18.55L7.6 5.45L9.93 5.45L14.5 18.55L12.02 18.55L11.11 15.74L6.37 15.74ZM8.75 8.43L7.13 13.44L10.37 13.44L8.75 8.43ZM16.74 18.36C15.99 18.07 15.39 17.71 14.93 17.26C15.01 17.03 15.65 15.2 15.73 14.98C16.17 15.39 16.69 15.72 17.29 15.97C17.9 16.22 18.5 16.34 19.1 16.34C19.77 16.34 20.27 16.22 20.59 15.97C20.91 15.7 21.07 15.36 21.07 14.94C21.07 14.63 20.97 14.37 20.78 14.17C20.59 13.96 20.35 13.79 20.05 13.67C19.76 13.54 19.36 13.4 18.85 13.25C18.08 13.02 17.44 12.78 16.94 12.54C16.45 12.31 16.02 11.93 15.66 11.4C15.31 10.88 15.13 10.18 15.13 9.31C15.13 8.54 15.29 7.86 15.62 7.25C15.94 6.62 16.42 6.13 17.06 5.77C17.71 5.41 18.5 5.23 19.44 5.23C20.09 5.23 20.73 5.33 21.35 5.53C21.97 5.73 22.51 6.01 22.98 6.39C22.91 6.62 22.33 8.46 22.25 8.69C21.31 8 20.37 7.66 19.42 7.66C18.76 7.66 18.27 7.8 17.95 8.07C17.64 8.35 17.48 8.71 17.48 9.16C17.48 9.6 17.66 9.94 18.02 10.17C18.39 10.38 18.95 10.59 19.7 10.8C20.48 11.04 21.11 11.28 21.61 11.51C22.11 11.75 22.53 12.12 22.88 12.64C23.24 13.15 23.42 13.84 23.42 14.71C23.42 15.46 23.25 16.15 22.92 16.77C22.6 17.38 22.12 17.87 21.46 18.23C20.81 18.59 20.02 18.77 19.09 18.77C18.28 18.77 17.5 18.64 16.74 18.36ZM27.56 15.24L27.56 18.55L25.21 18.55L25.21 5.45L27.56 5.45L27.56 11.57L32.08 5.45L34.71 5.45L30.47 11.29L34.95 18.55L32.2 18.55L28.93 13.4L27.56 15.24Z"
+            id="b2iXXQyQoY"
+        ></path>
+    </defs>
+    <g>
```

Comment:
- Can we remove the unused `<g>`?
- If it's not needed, can we also remove the `<defs>` above?

Once the changes are done, you will need to run the code through `svgo`. You can read more about it in the [contributions](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) guideline.